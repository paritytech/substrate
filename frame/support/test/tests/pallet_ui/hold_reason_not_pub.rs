#[frame_support::pallet]
mod pallet {
	#[pallet::config]
	pub trait Config: frame_system::Config {}

	#[pallet::pallet]
	pub struct Pallet<T>(core::marker::PhantomData<T>);

	#[pallet::composite_enum]
    enum HoldReason {}
}

fn main() {
}