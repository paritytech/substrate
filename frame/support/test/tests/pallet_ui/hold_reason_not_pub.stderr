error: Invalid pallet::composite_enum, `HoldReason` must be public
  --> tests/pallet_ui/hold_reason_not_pub.rs:10:5
   |
10 |     enum HoldReason {}
   |     ^^^^
